content/rs/networking/cluster-lba-setup.md

```diff
@@ -114,5 +118,5 @@ For Active Passive communication to work, you will need to expose database port(
 
```

Comment:
This article has 2 visible URL links under the "Intercluster communication considerations" section. These should probably be changed to relrefs instead.